webapp/src/globalContext/globalActions.ts

```diff
@@ -0,0 +1,22 @@
+import { components } from 'tg.service/apiSchema.generated';
+
+type OrganizationModel = components['schemas']['OrganizationModel'];
+type UsageModel = components['schemas']['UsageModel'];
+
+export type GlobalActionType =
```

Comment:
Maybe it would be better to avoid this redux style actions and use just functions? Is there any benefit of this approach I can't see?

Reply:
With separate functions it's difficult to make them stable (not causing re-renders).